### PR TITLE
feat: Button 컴포넌트 추가

### PR DESCRIPTION
**Pull request by hy57in**
## 📍 주요 변경사항

image

### CODE REVIEW

**Comment by hy57in** [comment]
displayname 
https://stackoverflow.com/questions/67992894/component-definition-is-missing-display-name-for-forwardref

**Comment by hy57in** [comment]
image

ㅇㅔ러 해결중..

**Comment by hy57in** [comment]
[af82648](https://github.com/sjoleee/bemyidol-frontend/pull/15/commits/af82648d41994d9944bdfd783469f9e287a4f176)
여기서 반영했습니다!

File: .eslintrc [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "prefer-const": "warn",
     "no-undef": "off",
     "import/order": [
-      "error",
+      "warn",
```
Comment (sjoleee) [inline .eslintrc:8]: LGTM!!

File: src/components/Layout/index.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ interface Props {
 }
 
 const Layout = ({ children }: Props) => {
-  return <div className="w-screen h-screen max-w-3xl m-auto shadow-md p-4">{children}</div>;
+  return <div className="w-screen max-w-3xl m-auto">{children}</div>;
```
Comment (sjoleee) [inline src/components/Layout/index.tsx:6]: 원래 코드가 더 최신인데 기존코드로 반영 부탁드려요

Comment (hy57in) [reply src/components/Layout/index.tsx]: [6ce0ee6](https://github.com/sjoleee/bemyidol-frontend/pull/15/commits/6ce0ee6420e10880db5148cbd0ec7881cf048bb3) 반영했습니다

**Review by sjoleee — APPROVED** [review]

**Pull request merged**